MultiplexReservationSystem/src/main/java/p/lodz/pl/multiplexreservationsystem/model/Reservations.java

```diff
@@ -0,0 +1,44 @@
+package p.lodz.pl.multiplexreservationsystem.model;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
+@EqualsAndHashCode
```

Comment:
Skoro taka ściana adnotacji to czemu nie `@Data`?

Reply:
@Data w lomboku oznacza:
- @ToString
- @EqualsAndHashCode
- @Getter/@Setter
- @RequiredArgsConstructor

Ja nie potrzebowałem ToStringa i RequiredArgsCtora tylko AllArgsCtora. Ostatecznie stwierdziłem, że lepszym rozwiązaniem jest klasa posiadająca tylko to, czego potrzebuje, niż oszczędzanie na adnotacjach i tym samym (niestety) czytelności.